homeassistant/components/miraie_ac/config_flow.py

```diff
@@ -0,0 +1,107 @@
+"""Config flow for MirAIe AC integration."""
+from __future__ import annotations
+
+import logging
+import re
+from typing import Any
+
+from py_miraie_ac import (
+    AuthException,
+    AuthType,
+    ConnectionException,
+    MirAIeAPI,
+    MobileNotRegisteredException,
+)
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.exceptions import HomeAssistantError
+
+from .const import CONFIG_KEY_PASSWORD, CONFIG_KEY_USER_ID, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONFIG_KEY_USER_ID): str,
+        vol.Required(CONFIG_KEY_PASSWORD): str,
```

Comment:
```suggestion
        vol.Required(CONF_PASSWORD): str,
```

Please use the available constants from `homeassistant.const`.